src/styles/tailwind.css

```diff
@@ -58,7 +58,19 @@ html {
   }
 
   .bootstrap3-grid {
-    @apply grid grid-cols-12 gap-30px;
+    @apply grid grid-cols-2 gap-30px;
+  }
+
+  @screen sm {
+    .bootstrap3-grid {
+      @apply grid grid-cols-6 gap-30px;
+    }
+  }
```

Comment:
since we are doing mobile first, should we set the styling for the sm screen as the initial styling for the grid?

Reply:
the styling for mobile is what we see above in lines 60 - 62 since it's the default so anything from 0px - 767px. 

Lines, 64 - 68 is saying, anything above sm (so anything 768px or above => @media (min-width: 768px) { ... }) will be tablet styling.